### PR TITLE
added Obsidian theme repo to the readme

### PR DESCRIPTION
**Pull request by mimishahzad**
image

### CODE REVIEW

**Comment by mimishahzad** [comment]
So far I've only got Rose Pine Moon, but the others are a wip at the moment!

File: readme.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ The community is encouraged to port Rosé Pine to their favourite apps 🌸
 - [Matplotlib](https://github.com/h4pZ/rose-pine-matplotlib) by [h4pZ](https://github.com/h4pz)
 - [Typora](https://github.com/rose-pine/typora)
 - [Zathura](https://github.com/edunfelt/zathura) by [edunfelt](https://github.com/edunfelt)
+- [Obsidian](https://github.com/mimishahzad/rose-pine-moon-obsidian)
```
Comment (mvllow) [inline readme.md:70]: Could we move this above Typora to keep it alphabetical

**Comment by mvllow** [comment]
Appreciate the contribution! Will get this merged as soon as the order is updated 😌

File: readme.md (hunk 1 above)
Comment (mvllow) [reply readme.md:70]: Also, please credit yourself, eg. add `by [you](your-github-url or similar)` to the end

**Comment by mimishahzad** [comment]
I opened a new PR from dev/mimi instead of the main branch 😄

**Pull request closed without merging**